OpenIPSL/Electrical/Renewables/PSSE/ElectricalController/REECA1.mo

```diff
@@ -0,0 +1,403 @@
+within OpenIPSL.Electrical.Renewables.PSSE.ElectricalController;
+model REECA1 "Electrical control model for large scale wind"
+  extends OpenIPSL.Electrical.Renewables.PSSE.ElectricalController.BaseClasses.BaseREECA(
+     Iqcmd(start=Iq0), Ipcmd(start=Ip0));
+
+  parameter OpenIPSL.Types.PerUnit Vdip = -99 "Low voltage threshold to activate reactive current injection logic (0.85 - 0.9)";
+  parameter OpenIPSL.Types.PerUnit Vup = 99 "Voltage above which reactive current injection logic is activated (>1.1)";
```

Comment:
Two things: 
 1. The description string requires/suggests (see next point) a range of 0.85 to 0.9 but the default is -99 totally outside the range. Why is that?
 2. Are these ranges (also for the remaining parameters) suggestions or requirements? If the latter then they should probably be also be implemented via `min` and `max` attributes.

Reply:
The range values come from the reports and manuals of the models. That being said, in practice when looking at parameter values in multiple PSSE .dyr file examples, what you realize is that they have parameters that exceed the allowable range. In this particular case, what I did was I utilized parameter data from an example that I got from while talking to Giuseppe. In this case, having Vdip = -99 and Vup = 99 is just a way of eliminating the max/min limits.

So I would say that they are suggestions more then anything (I just copied the description of the parameter as found in the reports, including the range).

Reply:
So maybe adding "typically" to those ranges?

Reply:
Like this? (Typically 0.85 - 0.9) ?

Reply:
yes, that should work.

Reply:
I don't know if you aware but no parameter is safe from the allowable limits. Sometimes we find "real life" examples that have the majority of its parameters out of range. How do we know this? Because PSSE displays the parameter in yellow when it's out of the "permissible" range. So what can be done? Should I write "typically" into ALL models that I implemented or maybe just remove the range in the description... Any suggestions?

Reply:
In order to be userfriendly, the ranges might be useful but at the same time, we then should have "typically" present as well to not confuse the user.

Reply:
Sounds good, I'll do the necessary modifications then.

Reply:
@fachif It looks like you have not yet done the modifications.